### PR TITLE
chore(test): delete test generated tmp file

### PR DESCRIPTION
**Pull request by fkuner**
I hereby agree to the terms of the CLA available at: https://databend.rs/dev/policies/cla/

## Summary

`Permission denied` if temporary files exist when different users execute `unit-test` and `stateless-test` in one machine.

## Changelog

- Build/Testing/CI

## Related Issues

Fixes #issue

### CODE REVIEW

File: tests/suites/0_stateless/20+_others/20_0007_http_clickhouse_handler.sh [hunk 1]
```diff
@@ -60,4 +60,4 @@ curl -s  -u 'root:' -XPOST "http://localhost:${QUERY_CLICKHOUSE_HTTP_HANDLER_POR
 curl -s  -u 'root:' -XPOST "http://localhost:${QUERY_CLICKHOUSE_HTTP_HANDLER_PORT}" -d "drop table a"
 
 
-# rm /tmp/*.txt
+rm /tmp/*.txt
```
Comment (Xuanwo) [inline tests/suites/0_stateless/20+_others/20_0007_http_clickhouse_handler.sh:63]: Seems dangerous to me...

Comment (fkuner) [reply tests/suites/0_stateless/20+_others/20_0007_http_clickhouse_handler.sh:63]: I think so too. We delete it one by one.

File: tests/suites/0_stateless/20+_others/20_0007_http_clickhouse_handler.sh [hunk 2]
```diff
@@ -60,4 +60,4 @@ curl -s  -u 'root:' -XPOST "http://localhost:${QUERY_CLICKHOUSE_HTTP_HANDLER_POR
 curl -s  -u 'root:' -XPOST "http://localhost:${QUERY_CLICKHOUSE_HTTP_HANDLER_PORT}" -d "drop table a"
 
 
-rm /tmp/*.txt
+rm /tmp/csv.txt /tmp/csv_names.txt /tmp/csv_names_and_types.txt /tmp/tsv_names_and_types.txt /tmp/values.txt
```
[the remark this answers is not in the document]
Comment (fkuner) [reply tests/suites/0_stateless/20+_others/20_0007_http_clickhouse_handler.sh:63]: Good idea.

**Review by Xuanwo — APPROVED** [review]

**Review by PsiACE — APPROVED** [review]

**Comment by Xuanwo** [comment]
sqllogictest failed for:

```shell
Parsed Statement
    at_line: 129,
    s_type: Statement: query, type: I, query_type: I, retry: False,
    suite_name: gen/03_dml/03_0020_group_by_negative,
    text:
        SELECT COUNT() FROM (SELECT (0 - number) :: Int64 AS c FROM numbers(65535) GROUP BY c);
    results: [(<re.Match object; span=(0, 4), match='----'>, 134, '65535')], statement type is query but return nothing
Traceback (most recent call last):
  File "main.py", line 33, in <module>
    fire.Fire(run)
  File "/home/runner/.local/lib/python3.8/site-packages/fire/core.py", line 141, in Fire
    component_trace = _Fire(component, args, parsed_flag_args, context, name)
  File "/home/runner/.local/lib/python3.8/site-packages/fire/core.py", line 466, in _Fire
    component, remaining_args = _CallAndUpdateTrace(
  File "/home/runner/.local/lib/python3.8/site-packages/fire/core.py", line 681, in _CallAndUpdateTrace
    component = fn(*varargs, **kwargs)
  File "main.py", line 29, in run
    http.run_sql_suite()
  File "/runner/_work/databend/databend/tests/logictest/logictest.py", line 347, in run_sql_suite
    self.execute()
  File "/runner/_work/databend/databend/tests/logictest/logictest.py", line 265, in execute
    self.batch_execute(statement_list)
  File "/runner/_work/databend/databend/tests/logictest/http_runner.py", line 25, in batch_execute
    self.execute_statement(statement)
  File "/runner/_work/databend/databend/tests/logictest/logictest.py", line 279, in execute_statement
    self.assert_execute_query(statement)
  File "/runner/_work/databend/databend/tests/logictest/logictest.py", line 310, in assert_execute_query
    f = format_value(actual, len(statement.s_type.query_type))
  File "/runner/_work/databend/databend/tests/logictest/logictest.py", line 203, in format_value
    width = len(str(vals[0])) + 2
IndexError: list index out of range
```

Seems not related to this PR? @ZeaLoVe Can you take a look at it? Thanks.

**Comment by fkuner** [comment]
> Is it ready for review or merge?

ready

**Comment by Xuanwo** [comment]
@mergify update

**Pull request merged**